src/Classes/TradeQuery.lua

```diff
@@ -464,13 +464,15 @@ function TradeQueryClass:UpdateControlsWithItems(slotTbl, index)
 	if errMsg == "MissingConversionRates" then
 		self:SetNotice(self.controls.pbNotice, "^4Price sorting is not available, falling back to DPS sort.")
 		sortedItems, errMsg = self:SortFetchResults(slotTbl, index, self.sortModes.DPS)
-	end
-	if errMsg then
+	elseif errMsg then
 		self:SetNotice(self.controls.pbNotice, "Error: " .. errMsg)
 		return
+	else
+		self:SetNotice(self.controls.pbNotice, "")
 	end
+
 	self.sortedResultTbl[index] = sortedItems
-	self.itemIndexTbl[index] = 1
```

Comment:
> I also included a fix fir the import button in #5375 but this seems to fix it differently.

Didn't check that PR yet but this line was wrong and had to be fixed. Looks like the intention was to reset the item index to 1 after sorting because dropdowns reset but the sorted item indexes are a little more convoluted than that. This is the correct way to reference the item on the first slot of dropdown containing sorted items.